drivers/telescope/lx200ap_experimental.cpp

```diff
@@ -120,12 +122,13 @@ bool LX200AstroPhysicsExperimental::initProperties()
     IUFillSwitch(&SwapS[1], "EW", "East/West", ISS_OFF);
     IUFillSwitchVector(&SwapSP, SwapS, 2, getDeviceName(), "SWAP", "Swap buttons", MOTION_TAB, IP_RW, ISR_1OFMANY, 0,
                        IPS_IDLE);
-
+#ifdef no
+    //2020-06-02, wildi, I'm not fond of this CMR sync
```

Comment:
Can you elaborate further on this?

Reply:
AP recommends to use :CM during the init phase and says explicitly not to use :CMR. :CMR is used to "re-calibrate" on a last slew while the mount is tracking. It does this even if the mount is on the "wrong side" of the pier. That means, the declination's axis points below horizon, as seen from the intersection of the hour with the declination axis in the direction of the telescope. :CMR needs a slew command before it can be used or as AP expresses it the object's co-ordinates needs to be set in memory. AP does not describe, what happens if used incorrectly. If I remember it correctly: a :CMR on the wrong side followed by goto command including a meridian flip will move the mount in the wrong direction. This behavior has to be tested with a physical mount. I recommend to leave it disabled until the test have been performed.